pywbem/cim_obj.py

```diff
@@ -253,6 +253,7 @@ class (the parent object) has a list of CIM properties, CIM methods and CIM
 
 from . import cim_xml
 from .config import DEBUG_WARNING_ORIGIN, SEND_VALUE_NULL
+import pywbem.config
```

Comment:
This should import from `.`, not from `pywbem`, to make sure it remains a relative import. In other words, this would become:
```
from . import config
```
and your new config variable would be accessed as:

```
config.IGNORE_NULL_KEY_VALUE
```